tests/Http/Offer/UpdateTypicalAgeRangeRequestHandlerTest.php

```diff
@@ -167,7 +167,9 @@ public function provideInvalidRequestBodies(): Iterator
             yield 'minimum age is bigger than maximum age ' . $offerType => [
                 'offerType' => $offerType,
                 'request' => '{ "typicalAgeRange": "12-6"}',
-                'expectedProblem' => ApiProblem::bodyInvalidData(),
+                'expectedProblem' => ApiProblem::bodyInvalidData(
+                    new SchemaError('/typicalAgeRange', '"from" age should not exceed "to" age')
```

Comment:
@bertramakers Don't know if it makes sense to make this a documented exception like we did with the main image selection?

Reply:
In this case I would go with `https://api.publiq.be/probs/body/invalid-data` because of the following reasons:

- Ranges in general need to start with a lower bound and end with a higher bound, that is not specific to our domain
- The data that causes the error is all included in the body (both the "from" & "to" ages), and editing the body is the only solution (= it's definitely invalid body data, there is no other fix)

Sometimes it's a grey zone though and only of those two reasons apply. In that case I tend to give the first reason (is it specific to us or also the case in general software) more weight when trying to decide about it.

Reply:
Makes sense, thanks for the explanation.